interface/modules/zend_modules/module/Application/view/layout/sendto.phtml

```diff
@@ -3,7 +3,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <?php echo $this->headTitle('ZF2 '. $this->translate('ZHOpenEMR'))->setSeparator(' - ')->setAutoEscape(false) ?>
+        <?php echo $this->headTitle('ZF2 '. $this->translate()->xl('OpenEMR'))->setSeparator(' - ')->setAutoEscape(false) ?>
```

Comment:
Wouldn't translate OpenEMR

Reply:
I'm assuming because we don't ever have different names for OpenEMR based on country/language?